### ISSUE TITLE
It seems that LuLu simply don't work 

### ISSUE THREAD

**Issue by Masroor69** [objective-see/LuLu#648]
I installed the Lu Lu app as mentioned in the documentation, on Sequoia 15.0.1.
add the network extensions and allow it to have control of the network. 
![Screenshot 2024-10-11 at 19 16 27](https://github.com/user-attachments/assets/4bd5ccff-f474-46b0-b093-141d4e4a8cf3)
and add the Telegram app to Block  List 
![Screenshot 2024-10-11 at 19 17 35](https://github.com/user-attachments/assets/1254d7ac-6a0c-4933-ad41-43e5bb66f426)

However, it seems that Telegram is still connecting and is able to transmit the data to its servers. 
![Screenshot 2024-10-11 at 19 18 34](https://github.com/user-attachments/assets/d156d766-7c8a-4df0-890f-e14d8ea3e595)
![Screenshot 2024-10-11 at 19 19 17](https://github.com/user-attachments/assets/d0554fb3-76d1-4061-8fe7-f9ff089deedd)
![Screenshot 2024-10-11 at 19 19 27](https://github.com/user-attachments/assets/b8f06256-775d-4ef8-acb1-030ffefc4e2e)
![Screenshot 2024-10-11 at 19 19 53](https://github.com/user-attachments/assets/386d79ac-770f-4353-b62a-5b22e3e5931d)

**Comment by mpas97** [comment]
Yeah, Apple messed everything up with macOS Sequoia. See eg. #643 where we are already discussing this behaviour. But it's not this plugin's fault if Apple decides to make the firewall behave like it's disabled and make all security plugin useless...

**Comment by Masroor69** [comment]
Yeap, I found that, if you **disable the system-level firewall** you can leverage the LULU.

**Comment by JStrategic** [comment]
Interestingly enough, I downloaded little snitch mini to see if it is encountering the same issue and little snitch seems to be working with the apple firewall enabled as well, not sure on the difference in how that application is handling the network filtering but thought I'd mention it.

**Comment by Masroor69** [comment]
One of the options I tested then was Little Snitch and it's not working with a system-level firewall. But it seems they solved the problem in the latest release as you said

**Comment by JStrategic** [comment]
Is Lulu working again for anyone after the 15.1 update? Still having issues on my end

**Comment by Masroor69** [comment]
@CoocooFroggy  @JStrategic 

you could enjoy using LULU if you disable the system-level firewall  at 
`preferences > network > firewall 
`
 until the patch comes

**Comment by dustookk** [comment]
same here， looking for patches

**Comment by objective-see** [comment]
These are (all?) Apple/macOS bugs ...some that yes, still exist in macOS 15.1

1. ["Apple’s new macOS Sequoia update is breaking some cybersecurity tools"](https://techcrunch.com/2024/09/19/apples-new-macos-sequoia-update-is-breaking-some-cybersecurity-tools/) 
2. ["Apple’s built-in macOS firewall breaks third-party firewalls"](https://www.obdev.at/blog/apples-built-in-macos-firewall-breaks-third-party-firewalls/)
3. ["A memory leak in Apple’s Network Extension framework"](https://www.obdev.at/blog/a-memory-leak-in-apples-network-extension-framework/)

Bug reports have been filed with Apple, so hopefully they'll fix the issue(s) soon? 🤞🏽

**Comment by JStrategic** [comment]
Wouldn't disabling the system-level firewall be a terrible idea, since that is responsible for handling incoming connections? Since lulu only handles the outgoing connections.

**Comment by JStrategic** [comment]
Lulu (2.9.7) is working for me now on MacOS 15.2! Just wanted to provide an update.
